artman/tasks/protoc_tasks.py

```diff
@@ -519,3 +519,24 @@ def execute(self, gapic_api_yaml):
                 continue
             if 'package_name' in go_settings:
                 return go_settings.get('package_name')
+
+
+class JavaProtoCopyTask(task_base.TaskBase):
```

Comment:
I was responsible for removing this as part of updating Java proto/grpc generation to run protoc in artman instead of in the generated gradle package. However, this had the side effect of no longer including the raw .proto files as part of the package, so this needs to be added back in.